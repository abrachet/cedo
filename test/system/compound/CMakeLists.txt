add_cedo_system_test(array_members.c array_members.cedo.c am)
add_cedo_system_test(bitfield.c bitfield.cedo.c bitfield)
add_cedo_system_test(class.cpp class.cedo.cpp c)
# add_cedo_system_test(end_padding.c end_padding.cedo.c two)
# add_cedo_system_test(middle_padding.c middle_padding.cedo.c two)
add_cedo_system_test(struct_members.c struct_members.cedo.c sm)
add_cedo_system_test(struct_test.c struct_test.cedo.c a)
# add_cedo_system_test(union.c union.cedo.c u)
