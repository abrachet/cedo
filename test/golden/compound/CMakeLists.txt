add_cedo_golden_test(CEDO_SRC basic_compound.cpp EXPECTED_OUTPUT basic_compound.s SYMS a)
add_cedo_golden_test(CEDO_SRC end_padding_struct.cpp EXPECTED_OUTPUT end_padding_struct.s SYMS a)
add_cedo_golden_test(CEDO_SRC mid_padding_struct.cpp EXPECTED_OUTPUT mid_padding_struct.s SYMS a)
